ApeWorX/ape-etherscan
# fix: correct URL in fantom API client

### What I did

The base URL for the api client when using Fantom networks was missing the `/api` suffix.
This PR fixes it as well as dusts off the plugin a little bit.

### How I did it

The issue was raised by the community so I addressed it!
I just debugged until I saw what was wrong and then fixed it and included fantom in the tests.

### How to verify it

You should now be able to get contract types from Fantom.

Use a script like this and connect to Fantom:

```python
from operator import ne
from ape import networks
import click


def main():
    provider = networks.provider
    network = provider.network
    assert network.ecosystem.name == "fantom", "Must use fantom"
    explorer = network.explorer
    assert explorer is not None, "Must have explorer (ape-etherscan) installed"
    address = "0x30Ac60fcbD79E03d51199BA87111b95C06e1E82F"
    contract_type = explorer.get_contract_type(address)
    click.echo(contract_type.name)
```

### Checklist
